components/picker/index.css

```diff
@@ -230,39 +228,23 @@ governing permissions and limitations under the License.
     pointer-events: none;
     content: '';
     position: absolute;
+    margin: calc(
+      (
+        var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) +
+        var(--mod-picker-focus-indicator-adjusted, var(--spectrum-picker-focus-indicator-adjusted)) +
+        var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness))
+      ) * -1);
     left: 0;
     right: 0;
     bottom: 0;
     top: 0;
-    margin: calc(
-      (var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) +
-       var(--mod-picker-border-width, var(--spectrum-picker-border-width)) *
-       -1)
-    );
-    inline-size: calc(
-      100% +
-      (var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) * 2) +
-      (var(--mod-picker-border-width, var(--spectrum-picker-border-width)) *2)
-    );
-    block-size: calc(
-      100% +
-      (var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) * 2) +
-      (var(--mod-picker-border-width, var(--spectrum-picker-border-width) * 2))
-    );
-    margin: auto;
-    margin-inline-start: calc(
-      (var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) +
-       var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness)) +
-       var(--mod-picker-border-size, var(--spectrum-picker-border-size))) *
-      -1
-    );
     border-style: solid;
     border-width: var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness));
     border-color: transparent;
     border-radius: calc(
       var(--mod-picker-border-radius, var(--spectrum-picker-border-radius)) +
       var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) +
-      var(--mod-picker-border-width, var(--spectrum-picker-border-width))
+      var(--spectrum-picker-border-width)
```

Comment:
You can `mod` this value, does it need to be included here incase they do? `--mod-picker-border-size`

Reply:
This also seems to be `size` instead of `width` are there two values or one?

Reply:
Good call-outs. We're refactoring this a bit, and that'll take care of both of these comments.